SciMaterialsActions.yml

```
@@ -0,0 +1,25 @@
+name: SciMaterialsProject
+
+on:
+  push:
+    branches: [ "dev" ]
+  pull_request:
+    branches: [ "dev" ]
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: 6.0.402
```

```suggestion
        dotnet-version: 6.0.x
```
Будет использоваться последняя версия

---

Я не убедился, что такой вариант будет запрашивать последнюю версию, поэтому поставил на данный момент актуальную 6.0.402, ок.